gen3/bin/config-env.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# This script will copy the environment configuration of the specified environment or apply
+# version updates to all the services and may specify particular values for a given subset of services
+
+source ${GEN3_HOME}/gen3/lib/utils.sh
+
+tgt_env=~/cdis-manifest/${USER}.planx-pla.net
+
+if [[ "$1" == "copy" ]]; then
+                git clone https://github.com/uc-cdis/${2}.git ~/temp_manifest
+                if [[ $? != 0 ]]; then
+        gen3_log_err "Something went wrong with getting source env check arguments"
```

Comment:
it'd be useful to add `https://github.com/uc-cdis/${2}.git` to this error log, to make it easier to understand what went wrong